### PR TITLE
Fix #24680 Remove Bootstrap and make base font size 16px

### LINKED ISSUE #24680: Remove Bootstrap Library in dotCMS
### Problem Statement
In the codebase of dotCMS we are importing the all the CSS for the Bootstrap library but we are only using the grid part in workflow portlets (it could be more places). 

This is causing a problem because the font sizing of bootstrap is interfering with our local font sizing and is hard to unify the base font size.

### Steps to Reproduce
1. Inspect a JSP portlet
2. Check the font size
3. See all the layers of font-sizing

### Acceptance Criteria
- The imported Bootstrap library should be removed from all parts of the system
- The affected parts should be fixed with custom CSS grids or flexbox
- The font sizing should be unified across all parts of the system
- Base font size 14px is mssing the buttons.

### External Links
N/A 

### Assumptions & Initiation Needs
- All parts of the system where the Bootstrap library is being used are identified
- The development team has the necessary knowledge to use custom CSS grids or flexbox

### Quality Assurance Notes
N/A

### PR DESCRIPTION
**Pull request by fmontes**
### Summary
🗑️🐛♻️

This pull request refactors and improves the SCSS styles for the backend UI, removing unused and redundant files and code, and applying a consistent grid system and design for the workflow portlet and the buttons. It also fixes a minor typo in the header box-shadow.

> _`_variables.scss`_
> _Refactoring styles for buttons_
> _Autumn of cleanup_

### Walkthrough
*  Remove unused screen size variables from `_variables.scss` ([link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-2728d896cd6de95b52033ed77ebb0e002a807f786c31802032738931ea1c462bL76-L80))
*  Move dijit and fake dojo button styles from `dotcms.css` to `_variables.scss` for consistency and accessibility ([link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-e54faf6cc7fa19f096fa91aa75cb3beab88581eee9cedc1a571693d225db5b02L5041-R5149))
*  Remove duplicated dijit and fake dojo button styles from `dotcms.css` ([link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-e54faf6cc7fa19f096fa91aa75cb3beab88581eee9cedc1a571693d225db5b02L7084-L7192))
*  Fix typo in box-shadow property of `#hd` selector in `_reset-fonts-grids.scss` ([link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-a4efb3cc0c8ee7711302cd0884367947b948c031361a80238d2ab45d6b34aa11L132-R132))
*  Update layout of `.view-actions` container in workflow portlet using grid system and adding margins and gaps in `_view-workflow.scss` ([link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-1c94baf61f6cbba3f7a6fe38bb904451ea081bd9b521b25cdb4cf7edb02a667bL28-R35))
*  Remove unnecessary styles for `.view-actions__permissions` selector in `_view-workflow.scss` ([link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-1c94baf61f6cbba3f7a6fe38bb904451ea081bd9b521b25cdb4cf7edb02a667bL55-R55))
*  Add styles for `.view-actions__sub-actions` and `.view-actions__workflow-actions` selectors to separate them from main content in `_view-workflow.scss` ([link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-1c94baf61f6cbba3f7a6fe38bb904451ea081bd9b521b25cdb4cf7edb02a667bR67-R74))
*  Delete unused bootstrap files from `bootstrap-sass-3.3.7` folder ([link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-7e9a28f5eed4b43ad5546bbf7272439df64f903d57dfa0bfc1f07c167ec41711), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-7fa17e912e68425f234c26311f15d5b76a9ad2cea08aa2dde0a3ea3abf6a26e8), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-65b8b8d24356b3ae3e5d67d2f030a504d73a322246a5321368c634e847dd188f), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-1d55a52ab1a76a88919d7a3e29bd3181be35bf6f57252c12abffcded69703cde), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-fd84a129775c878cb6ce65133305eb078d983e24df9a9c774ee441e54a3178bd), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-75a53a5dabfa870969c518c8126ebde2e43eccb90b9babd21b6649e8e2565b6b), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-9922f255e574b0fabfa5288bb06a50fa009046f45ab7e492cad72bec56b823f6), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-ed0575265bcf0a33a69dbbf8ee4e0b38602b40b66c0b5383fc86ceade64b20bd), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-54a78b6d5833f4bcca0e3dc1752bf69c0113ecf38712095e1ab7386d393e59fa), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-799f06cacc05dab8e846cd42c7453bf29fbaf3106955b8abf123d124296e9cfe), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-737c50c57459908d1469ec1ec4b5b905170106e111e1a417cc0000fd001f8973), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-d5356d867ca58fda9f5fc4d0a1558e35583d9fd0e8cfd5f39aff0081a9d6ef5d), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-f8dc7d83ad685e3c90a91caaaf1da9ca41c96c75c5be4e86486d5fb3a58ffa80), [link](https://github.com/dotCMS/core/pull/25817/files?diff=unified&w=0#diff-12421b050f586af5b8253d4f44c4f9cb5b6f0138b735effad7ab334c749bfeec))

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Review by zJaaal — APPROVED** [review]
LGTM

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.scss [hunk 1]
```diff
@@ -34,7 +34,7 @@ p-inputSwitch {
             @include md-icon;
             color: $color-palette-primary;
             content: "lock_open";
-            font-size: $font-size-default;
+            font-size: $font-size-sm; // 14px
```
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.scss:37]: shouldn't be `$font-size-md`?

File: core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-notifications/components/dot-notifications/dot-notifications-item.component.scss [hunk 2]
```diff
@@ -41,7 +41,7 @@ a {
 }
 
 .notification-item__title {
-    font-size: $font-size-default;
+    font-size: $font-size-sm;
```
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-notifications/components/dot-notifications/dot-notifications-item.component.scss:44]: same question.

File: core-web/libs/dotcms-scss/angular/dotcms-theme/_misc.scss [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 .p-component {
     font-family: $font-default;
-    font-size: $font-size-default;
+    font-size: $font-size-md;
```
Comment (zJaaal) [inline core-web/libs/dotcms-scss/angular/dotcms-theme/_misc.scss:5]: isn't `$font-size-default` the same as `$font-size-md`?

File: core-web/libs/dotcms-scss/jsp/scss/backend/_common.scss [hunk 4]
```diff
@@ -37,9 +37,13 @@
...
 .input-text-naked {
-    border: none;
-    color: $font-color-base;
+    background-color: $color-palette-gray-100;
+    border: solid 1px $color-palette-gray-200;
+    border-radius: $border-radius-md;
+    color: $color-palette-gray-700;
     font-family: $font-default;
-    font-size: $font-size-default;
+    font-size: $font-size-md;
```
Comment (zJaaal) [inline core-web/libs/dotcms-scss/jsp/scss/backend/_common.scss:45]: same

File: core-web/libs/dotcms-scss/jsp/scss/backend/_common.scss [hunk 5]
```diff
@@ -37,9 +37,13 @@
 }
 
 .input-text-naked {
-    border: none;
-    color: $font-color-base;
+    background-color: $color-palette-gray-100;
+    border: solid 1px $color-palette-gray-200;
```
Comment (zJaaal) [inline core-web/libs/dotcms-scss/jsp/scss/backend/_common.scss:41]: this should be 1.5px.

**Review by zJaaal — CHANGES_REQUESTED** [review]
Where are we setting the new font size to the root?

**Review by hmoreras — APPROVED** [review]

**Pull request merged**